velox/common/memory/MemoryUsageTracker.cpp

```diff
@@ -26,10 +33,16 @@ std::shared_ptr<MemoryUsageTracker> MemoryUsageTracker::create(
 }
 
 MemoryUsageTracker::~MemoryUsageTracker() {
-  VELOX_CHECK_EQ(usedReservationBytes_, 0);
-  VELOX_CHECK_EQ(reservationBytes_, 0);
-  VELOX_DCHECK_EQ(grantedReservationBytes_, 0);
-  VELOX_DCHECK_EQ(minReservationBytes_, 0);
+  VELOX_DCHECK_EQ(usedReservationBytes_, 0);
+  if (usedReservationBytes_ != 0) {
+    LOG_EVERY_N(ERROR, kLogEveryN)
+        << "usedReservationBytes_ is not zero " << toString();
```

Comment:
Do we need to show the stack trace as well to locate the tracker instance, or there is usually only one and we don't need to do so?

Reply:
Good point. I will try to add printStackTrace() for such non-exception case.